src/contour/helper.h

```diff
@@ -133,7 +133,7 @@ constexpr inline vtbackend::Modifiers makeModifiers(Qt::KeyboardModifiers qtModi
     // that are missing the AltGr key.
     // Microsoft does not recommend using Ctrl+Alt modifier for shortcuts.
     auto constexpr AltGrEquivalent = Modifiers { Modifier::Alt, Modifier::Control };
-    if (modifiers.contains(AltGrEquivalent))
+    if (modifiers.containsAll(Modifier::Alt, Modifier::Control))
```

Comment:
With the comment below, I think we should be able to revert this line again. :-)